geoslurp_userplugins/sword.py

```diff
@@ -0,0 +1,84 @@
+# This file is part of geoslurp.
+# geoslurp is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+
+# geoslurp is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+
+# You should have received a copy of the GNU Lesser General Public
+# License along with geoslurp; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+
+# Author Roelof Rietbroek (r.rietbroek@utwente.nl), 2023
+# Author Amin Shakya (a.shakya@utwente.nl), 2023
+
+from geoslurp.dataset.OGRBase import OGRBase
+from geoslurp.datapull.http import Uri as http
+from geoslurp.datapull.github import Crawler as ghCrawler
```

Comment:
The ghcrawler and other github related imports are not needed here. I guess these are leftovers from the original file